### ISSUE TITLE
[FEATURE REQUEST] Dropout layer

### ISSUE THREAD

**Issue by LdDl** [LdDl/cnns#4]
**Is your feature request related to a problem? Please describe.**
See ref. https://en.wikipedia.org/wiki/Dilution_(neural_networks)
It should be applied to neural network only when it's in learning stage.

**Describe the solution you'd like and provide pseudocode examples if you can**
Just use existing best practices for this layer.
Pytorch may be? https://pytorch.org/docs/master/generated/torch.nn.Dropout.html

**Describe alternatives you've considered and provide pseudocode examples if you can**

**Additional context**
nope
